tests/test_throttling.py

```diff
@@ -9,49 +6,6 @@
 from .entities import AuthorizedUser
 
 
-@pytest.mark.parametrize('need_auth', (True, False))
-def test_throttling(need_auth):
-    client = APIClient()
-
-    if need_auth:
-        user = AuthorizedUser()
-        client.force_authenticate(user)
-
-    for i in range(1, 5):
-        response = client.get('/with-throttling-on-controller/')
-        response_of_same = client.get('/with-throttling-on-controller/same/')
-
-        if i > 6 // 2:
-            assert response.status_code == response_of_same.status_code == HTTPStatus.TOO_MANY_REQUESTS, i
-        else:
-            assert response.status_code == response_of_same.status_code == HTTPStatus.OK, i
-
-
-def test_throttling_discards_old_requests():
-    if sys.version_info > (3, 8):
-        return
-
-    from freezegun import freeze_time
-
-    client = APIClient()
-    user = AuthorizedUser()
-    client.force_authenticate(user)
-
-    with freeze_time(datetime.datetime(2000, 1, 1, second=0, microsecond=0)):
-        for _ in range(3):
-            client.get('/with-throttling-on-controller/')
-
-    with freeze_time(datetime.datetime(2000, 1, 1, second=0, microsecond=600000)):
-        for _ in range(2):
-            client.get('/with-throttling-on-controller/')
-
-    # Act
-    with freeze_time(datetime.datetime(2000, 1, 1, second=1, microsecond=0)):
-        response = client.get('/with-throttling-on-controller/')
-
-    assert response.status_code == HTTPStatus.OK
-
-
 @pytest.mark.parametrize('need_auth', (True, False))
 def test_throttling_on_method(need_auth):
```

Comment:
Rename to `test_throttling` as it's obvious that throttling is configured for a method.